### PR TITLE
29089 signature for agreements

### PR DESCRIPTION
**Pull request by jano-omugtong**

### CODE REVIEW

File: app/src/main/java/com/creditsaison/loansystem/fragments/LoanAgreementFragment.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+        return binding.getRoot();
+
+
+    }
+
+    @Override
+    public void onResume(){
+        super.onResume();
+        Log.v("log_tag", "uyNagResume");
```
Comment (jeremiahllanto) [inline app/src/main/java/com/creditsaison/loansystem/fragments/LoanAgreementFragment.java]: Remove all Log.v()

File: app/src/main/java/com/creditsaison/loansystem/fragments/AccountNewFragment.java [hunk 2]
```diff
@@ -1,47 +1,195 @@
...
+            client_image = (ImageView) binding.getRoot().findViewById(R.id.client_image);
+
+            btn_add_photo.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    captureImage();
+                }
+            });
+
+//            btn_add_photo.setOnClickListener(new View.OnClickListener() {
```
Comment (jeremiahllanto) [inline app/src/main/java/com/creditsaison/loansystem/fragments/AccountNewFragment.java:87]: Remove blocks of code that are commented out

File: app/src/main/java/com/creditsaison/loansystem/fragments/SignatureFragment.java [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+                    Log.v("log_tag", "dito ba?");
+
+
+                    Log.v("log_tag", currentPage);
+
+                    Log.v("log_tag", v.toString());
+
+                    if (currentPage == "termsCondition"){
+                        Navigation.findNavController(v).navigate(R.id.action_signatureFragment_to_termsConditionFragment);
+//
```
Comment (jeremiahllanto) [inline app/src/main/java/com/creditsaison/loansystem/fragments/SignatureFragment.java]: Remove block of code

File: app/src/main/java/com/creditsaison/loansystem/fragments/AccountNewFragment.java (hunk 2 above)
Comment (jano-omugtong) [reply app/src/main/java/com/creditsaison/loansystem/fragments/AccountNewFragment.java:87]: this is out of the current scope for the branch

File: app/src/main/java/com/creditsaison/loansystem/viewmodel/AccountNew2ViewModel.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+public class AccountNew2ViewModel extends ViewModel {
+
+    public AccountNew2ViewModel() {
+
+    }
+
+    public void onButtonClick(View view) {
+        switch (view.getId()) {
+            case R.id.btn_submit:
+                Log.v("log_tag", "nag case sya");
```
Comment (jeremiahllanto) [inline app/src/main/java/com/creditsaison/loansystem/viewmodel/AccountNew2ViewModel.java:20]: Remove Log.v()

File: app/src/main/java/com/creditsaison/loansystem/viewmodel/AccountNew2ViewModel.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+                break;
+
+            default:
+                Log.v("log_tag", "default");
```
Comment (jeremiahllanto) [inline app/src/main/java/com/creditsaison/loansystem/viewmodel/AccountNew2ViewModel.java:24]: Remove Log.v()

File: app/src/main/java/com/creditsaison/loansystem/viewmodel/AccountNew2ViewModel.java [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+                break;
+
+        }
+        Log.v("log_tag", "para lang =malaman ang view");
```
Comment (jeremiahllanto) [inline app/src/main/java/com/creditsaison/loansystem/viewmodel/AccountNew2ViewModel.java:28]: Remove Log.v()

File: app/src/main/java/com/creditsaison/loansystem/viewmodel/AccountNew2ViewModel.java [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+        Log.v("log_tag", view.toString());
```
Comment (jeremiahllanto) [inline app/src/main/java/com/creditsaison/loansystem/viewmodel/AccountNew2ViewModel.java:29]: Remove Log.v()

File: app/src/main/java/com/creditsaison/loansystem/viewmodel/AccountNew2ViewModel.java [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+        Log.v("log_tag", "para lang malaman ang view");
```
Comment (jeremiahllanto) [inline app/src/main/java/com/creditsaison/loansystem/viewmodel/AccountNew2ViewModel.java:30]: Remove Log.v()

File: app/src/main/java/com/creditsaison/loansystem/fragments/SignatureFragment.java [hunk 9]
```diff
@@ -0,0 +1,321 @@
...
+                    editor.putString("promNoteImg", StoredPath);
+                }
+
+                editor.commit();
+
+                mFileOutStream.flush();
+                mFileOutStream.close();
+
+            } catch (Exception e) {
+                Log.v("log_tag", e.toString());
```
Comment (jeremiahllanto) [inline app/src/main/java/com/creditsaison/loansystem/fragments/SignatureFragment.java:201]: Is this needed?

File: app/src/main/java/com/creditsaison/loansystem/fragments/SignatureFragment.java [hunk 10]
```diff
@@ -0,0 +1,321 @@
...
+
+            lastTouchX = eventX;
+            lastTouchY = eventY;
+
+            return true;
+        }
+
+        private void debug(String string) {
+
+            Log.v("log_tag", string);
```
Comment (jeremiahllanto) [inline app/src/main/java/com/creditsaison/loansystem/fragments/SignatureFragment.java:263]: Is this needed?

File: app/src/main/java/com/creditsaison/loansystem/viewmodel/AccountNew2ViewModel.java (hunk 8 above)
Comment (jeremiahllanto) [reply app/src/main/java/com/creditsaison/loansystem/viewmodel/AccountNew2ViewModel.java:30]: out of scope

File: app/src/main/java/com/creditsaison/loansystem/viewmodel/AccountNew2ViewModel.java (hunk 7 above)
Comment (jeremiahllanto) [reply app/src/main/java/com/creditsaison/loansystem/viewmodel/AccountNew2ViewModel.java:29]: out of scope

File: app/src/main/java/com/creditsaison/loansystem/viewmodel/AccountNew2ViewModel.java (hunk 6 above)
Comment (jeremiahllanto) [reply app/src/main/java/com/creditsaison/loansystem/viewmodel/AccountNew2ViewModel.java:28]: out of scope

File: app/src/main/java/com/creditsaison/loansystem/viewmodel/AccountNew2ViewModel.java (hunk 5 above)
Comment (jeremiahllanto) [reply app/src/main/java/com/creditsaison/loansystem/viewmodel/AccountNew2ViewModel.java:24]: out of scope

File: app/src/main/java/com/creditsaison/loansystem/viewmodel/AccountNew2ViewModel.java (hunk 4 above)
Comment (jeremiahllanto) [reply app/src/main/java/com/creditsaison/loansystem/viewmodel/AccountNew2ViewModel.java:20]: out of scope

**Pull request merged**